eRecruiter.Api/Responses/Mandator/MandatorResponse.cs

```diff
@@ -1,14 +1,38 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
 
 namespace eRecruiter.Api.Responses
 {
     public class MandatorResponse
     {
+        /// <summary>
+        /// The unique identifier of the mandator.
+        /// </summary>
         public int Id { get; set; }
+
+        /// <summary>
+        /// The name of the mandator.
+        /// </summary>
         public string Name { get; set; }
+
         public PortalSettingsResponse PortalSettings { get; set; }
+
+        /// <summary>
+        /// All available translations for values, identified by a key.
+        /// </summary>
         public IEnumerable<TranslationResponse> Translations { get; set; }
+
+        /// <summary>
+        /// A list containing all the job profile names.
+        /// </summary>
+        [Obsolete("Use GroupedJobProfiles instead")]
```

Comment:
I don't see the old property as obsolete. Most users of the API will never need the grouped version of the job profiles, so most users will use the "old" flat list. I think the `Obsolete` attribute is not necessary.